server/service/integration_mdm_lifecycle_test.go

```diff
@@ -780,6 +800,11 @@ func (s *integrationMDMTestSuite) TestLifecycleSCEPCertExpiration() {
 	err = RenewSCEPCertificates(ctx, logger, s.ds, &fleetCfg, s.mdmCommander)
 	require.NoError(t, err)
 
+	mysql.ExecAdhocSQL(t, s.ds, func(q sqlx.ExtContext) error {
+		mysql.DumpTable(t, q, "nano_cert_auth_associations")
+		return nil
```

Comment:
Weidly (at least based on my understanding), one row always seems to have a NULL renewal command uuid at this point (I would've expected all rows to have a renewal command?)